build.gradle

```diff
@@ -21,7 +21,7 @@ buildscript {
     }
     dependencies {
         classpath "cz.alenkacz:gradle-scalafmt:${gradle.scalafmt.version}"
-        classpath 'org.scoverage:gradle-scoverage:2.1.0'
+        classpath 'org.scoverage:gradle-scoverage:2.3.0'
```

Comment:
Also checked that there is no new release of [scalac-scoverage-runtime][1] (we use 1.3.1 which is current latest stable release)

[1]: https://github.com/scoverage/scalac-scoverage-plugin/releases